frontend/config/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (1.0.1).
```

Comment:
Do we mock service workers for storybook? As far as I know we don't use them for the app.

Reply:
yes, this is just used for storybook. I dont think it will run the mocks in dev/prod, b/c the service worker is only initiated in the .storybook/preview.js file